src/viewer.js

```diff
@@ -813,10 +813,15 @@ Viewer.prototype = {
   },
 
   ballsAndSticks : function(name, structure, opts) {
+    if (opts.radius) {
```

Comment:
This currently fails in case no options are passed to the ballsAndSticks function. Apart from this, IMHO, the better approach would be to override both cylRadius and sphereRadius in case the radius option is provided, e.g.

```js
options.cylRadius = options.radius || options.cylRadius || 0.1;
options.sphereRadius = options.radius || options.sphereRadius || 0.3;
```